src/flowchem/components/valves/valve.py

```diff
@@ -219,8 +219,9 @@ async def get_position(self) -> tuple[tuple, tuple]:
         pos = int(pos) if pos.isnumeric() else pos
         return self._positions[int(self._change_connections(pos, reverse=True))]
 
-
-    async def set_position(self, connect: str | tuple = "", disconnect: str | tuple = "",
+    async def set_position(self,
+                           connect: str = "",
```

Comment:
is it really required to have a default argument here?

Reply:
Yes
If we remove the default argument here, this argument will not appear in the API server.